styles/styles.css

```diff
@@ -91,6 +92,49 @@ header a {
   height: var(--hamburger-icon-height);
 }
 
+.mobile-menu {
+  display: none;
+  position: absolute;
```

Comment:
- [ ] It would be better to change the position of the mobile menu to fixed so then when we scrolled we can't see the other elements the menu stays fixed 🙃 

Position: absolute 

https://github.com/jecode93/portfolio/assets/74501523/43196c5b-1422-4432-a78b-250dea187300

Position: fixed

https://github.com/jecode93/portfolio/assets/74501523/274360dd-2246-4ec5-a5a3-151d89a1f210